### ISSUE TITLE
RouterClient.Cost name confusing

### ISSUE THREAD

**Issue by Lotterleben** [Lotterleben/AODVv2-Draft#5]
From Jiazi's review:

[Jiazi]
> 4.2 router client set
>
>>    RouterClient.Cost
>>       The cost associated with reaching this address or address range.
>
>
> Both "metric" and "cost" are used in the draft. If they are the same,
> please pick a term and use it consistently.
>

[Lotte]

> Cost refers to the result of the Cost(L) function for the link to the
> RouterClient. Would it make more sense it this entry was called LinkCost?
>

[Jiazi]
It's still a metric, no? Do we need to have a difference here?

**Comment by charliep51** [comment]
Hello folks,

A cost metric is a good metric.  I thought we had to support metrics 
that were not cost metrics.

However, even if we only supported cost metrics, I think that using 
language that is easy to understand is more important than getting rid 
of either the term metric or the term cost.  We should keep the term 
metric for sure.  We should keep the term cost if it is convenient and 
makes the document read well.

I am O.K. with defining a new function called "LinkCost" if anyone wants 
to do that.

Regards,
Charlie P.
